wix-incubator/pro-gallery
# [SP] consolidate oneRow with scrollDirection

* replace oneRow with scrollDirection

* fix import in styles migrator

* fix to styleMigrator

* more removal of onerow

* more

* refactor for layouter. styleMigrator there

* export stylesMigrator

* export stylesMigrator

* dummy

* import stylesMigrator

* change oneRow SP unitTests to scrollDirection

* hasThumbnails tests

* remove oneRow in gallery unittests. make them pass

* leave oneRow e2e to make sure this whole change isnt breaking

* add deprecation warning and error in is relevant.

* test text

* remove oneRow in processLayouts!!!! YES!

* add deprecation text to integration e2e tests that make sure we dont break users

* remove unused import

* add test link frrom lib to layouts

* fix stylesMigrator function

* fix stylesMigrator affected by global replaces

* change oneRow's default value to undefined as its deprecated

* keep oneRow in the styleParams object. So it wont break outgoing events API

* make it breaking

* add a migration guide(/log)

* snapshots for scrollDirection
